src/main/java/gregtech/common/metatileentities/multi/MetaTileEntityLargeBoiler.java

```diff
@@ -562,14 +556,13 @@ public boolean onRightClick(EntityPlayer playerIn, EnumHand hand, EnumFacing fac
         }
         for (int slotIndex = 0; slotIndex < itemImportInventory.getSlots(); slotIndex++) {
             ItemStack itemStack = itemImportInventory.getStackInSlot(slotIndex);
-            int burnTime = (int) Math.ceil(TileEntityFurnace.getItemBurnTime(itemStack) / (50.0 * boilerType.fuelConsumptionMultiplier * getThrottleMultiplier()));
+            int burnTime = (int) Math.ceil(TileEntityFurnace.getItemBurnTime(itemStack) / (50.0 * boilerType.fuelConsumptionMultiplier)) * getHeatEfficiencyMultiplier() * currentTemperature == boilerType.maxTemperature ? 2 : 1;
```

Comment:
Why the change away from the Throttle Multiplier to determine the burn time, and why does the fuel burn longer when at maximum temperature?

Reply:
Why does the fuel burn longer at max temp ? I don't know it worked like that before I changed how the boilers works ...
If I am not wrong this code is acting on the the one probe integration so that's just a fix of the showed burn time not taking this behavior in account explaining why I removed the throttle from the computation as I changed this part of how the throttle works